src/bufr/README.md

```diff
@@ -116,6 +116,11 @@ ioda encoder. It has the following sections:
       `minute`, _(optional)_ `second`, and _(optional)_ `hoursFromUtc` (must be an **integer**).
       Internally, the value stored is number of seconds elapsed since a reference epoch, currently
       set to 1970-01-01T00:00:00Z.
+    * `timeoffset` Associate **key** with data for mnemonic for `timeOffset`, that should result
+      in seconds relative to an ISO-8601 string of date and time (e.g., `2020-11-01T11:42:56Z`).
+      If the timeOffset mnemonic is a floating-point value in hours, then simply use **transform**
+      and scale by 3600 seconds.  Internally, the value stored is number of seconds elapsed since
+      a reference epoch, currently set to 1970-01-01T00:00:00Z.
```

Comment:
Might want to add example usage in sample yaml section above.

Reply:
OK, I did.